### PR TITLE
Smoother Cult Reincarnation Animation

### PR DESCRIPTION
**Pull request by gurfan**
## What this does
Just adds a tiny animation showing the shade morphing into a more human appearance.

## Why it's good
Looks nicer.

## Changelog
This is so minor it doesnt need one.

### CODE REVIEW

**Comment by PrimeDSS13** [comment]
post it

File: code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm [hunk 1]
```diff
@@ -2891,13 +2892,13 @@ var/list/bloodcult_exitportals = list()
 			vessel.my_appearance.b_eyes = 21
 			vessel.my_appearance.s_tone = 45 // super duper albino
 
+			vessel.equip_or_collect(new /obj/item/clothing/under/rags(vessel), slot_w_uniform)
```
Comment (Kurfursten) [inline code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm]: why is this order of operations being changed?

Comment (Kurfursten) [reply code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm]: I don't think it probably matters, but this was undocumented and I want to know why

Comment (gurfan) [reply code/datums/gamemode/factions/bloodcult/bloodcult_runespells.dm]: I was originally changing something else but decided not to include it in this PR. I've fixed the order of operations.

**Pull request merged**